src/core/lib/gprpp/atomic.h

```diff
@@ -21,10 +21,83 @@
 
 #include <grpc/support/port_platform.h>
 
-#ifdef GPR_HAS_CXX11_ATOMIC
-#include "src/core/lib/gprpp/atomic_with_std.h"
-#else
-#include "src/core/lib/gprpp/atomic_with_atm.h"
-#endif
+#include <atomic>
+
+namespace grpc_core {
+
+template <typename T>
+using Atomic = std::atomic<T>;
```

Comment:
API suggestion: Instead of defining this as `std::atomic<>` and then requiring callers to use standalone functions instead of the methods that are exposed by `std::atomic<>`, would it make sense to instead make this a class containing a single `std::atomic<>` data member and providing the methods we want to expose?

Reply:
That's a great idea. Thank you. PTAL.